airbyte-workers/build.gradle

```diff
@@ -77,3 +77,22 @@ task copyGeneratedTar(type: Copy) {
 Task dockerBuildTask = getDockerBuildTask("worker",  "$project.projectDir")
 dockerBuildTask.dependsOn(copyGeneratedTar)
 assemble.dependsOn(dockerBuildTask)
+
+
+test {
+    useJUnitPlatform {
+        excludeTags 'log4j2-config', 'cloud-storage'
+    }
+    testLogging {
+        events "passed", "skipped", "failed"
+    }
+}
+
+task cloudStorageIntegrationTest(type: Test) {
```

Comment:
Nit, we can update the gradle.yaml file for in the .github folder to run those test in github action as well (similar to the integration tests)

Reply:
so these tests require quite a big of environment setup (see: cloud_storage_logging_test.sh). is that compatible with the approach you are describing?

Reply:
Yes, we can make sure that it only run on a release build or on a scheduled build using the same condition that has been use for the slow integration test or move those test as slow integration test and make the environment setup as a part of the slow integration test setup.

Reply:
what's the benefit to doing that?

Reply:
Making sure that the test is run before releasing without having too much impact on the build time for regular builds.

Reply:
understand. the tricky thing here is that these tests function as unit tests today, since testing against actual buckets is the recommended thing to do. if we move this to master/release only tests, we'll lose testing coverage here.

was there a specific time unit you want to meet? or was this suggestion opportunistic?

Reply:
seems like there are some open questions here. i don't think this thread should block merging as it is leveraging an existing pattern.